Main.cs

```diff
@@ -1,93 +1,211 @@
 using System;
 using System.Collections.Generic;
 
-// Author: Sonia Álvarez
-
 class Program
 {
+    // Nota de Sonia:
+        // El método Main() debe ser estático en .NET (creo)
     static void Main()
     {
-        // Probando que compila y que se usan tipos enumerados
-       const int fuerzaAldeana = 0;
-       const int fuerzaGuerrero = 10;
-       const int fuerzaArquero = 5;
+        // Constantes de fuerza de cada unidad
+        const int fuerzaAldeana = 0;
+        const int fuerzaGuerrera = 10;
+        const int fuerzaArquera = 5;
+        // Constante de salud/vida
+        const int vida = 20;
+
+        // Antes de la primera elección de turno
+        int equipoAtacante = -1;
```

Comment:
**Duda:** ¿Podéis explicarme esta inicialización a -1?

Reply:
Se inicializa a +-1 porque se entiende LanzarAtaque por turnos. El primer turno es como una monea al aire y desde ahí, se alternan turnos 0 a 1 a 0 ... etc